### PR TITLE
Divider

### LINKED ISSUE #647: [COMPONENT] Divider
**Describe the solution you'd like**
A divider is a thin line that groups content in lists, layouts and items.

We would like two different versions and the behavior is like on our cards:
- onWhite
- onLight
- onDark (Not yet but will come eventually)

**Describe alternatives you've considered**
We have talked about the component before, but back then it was not worth

**Additional context**
See also examples here: 

https://design-system.futurelearn.com/atoms/divider
https://material.io/components/dividers/

in zeplin they look like this:
onWhite: https://zpl.io/a31EO51
onLight: https://zpl.io/blD3xq1

### Kick Off:
- [x] Accept Feature incl. name of component (DSL)
- [x] Update Kirby Component Status and publish to [Kirby Cookbook](https://cookbook.kirby.design)
### Code:
- [x] Create Feature Branch
- [x] Implement Skeleton Component
- [x] Add any required mock and fakes to the `KirbyTestingModule`
- Ask UX for review:
    - [x] Ensure implementation is correct in relation to UX design
    - [x] Define the version of the implementation + steps needed to reach version 1.0 (create [enhancement requests](https://github.com/kirbydesign/designsystem/issues/new?assignees=&labels=enhancement&template=enhancement-request.md&title=%5BEnhancement%5D))
- [x] [Implement Examples and Showcase](https://cookbook.kirby.design/home/showcase/button)
- [x] Add link to showcase on Kirby Component Status (`component-status-items.ts => code.cookbookUrl`)
- [x] Also remove `githubIssueNo` and set `status` to `ItemCodeStatus.ready`
- [x] Push commits to GitHub
### Review:
- [x] Open a pull request (remember to add `closes #issueno` to the description) and ask @kirbydesign/kirby-guild for review
- Merge to master and delete branch

### PR DESCRIPTION
**Pull request by MortenGregersen**
Closes #647

### CODE REVIEW

File: src/app/examples/chip-example/chip-example.component.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, Input } from '@angular/core';
```
Comment (jakobe) [inline src/app/examples/chip-example/chip-example.component.ts]: Revert, not related

File: src/kirby/components/divider/divider.component.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import '../../scss/utils';
+
+hr {
+  border: 0px;
+  border-top: 1px solid var(--kirby-divider-color);
+
+  :host-context(.default),
```
Comment (jakobe) [inline src/kirby/components/divider/divider.component.scss]: Usualt applied the other way around:
`:host-context(.default) hr`

File: src/app/examples/divider-example/divider-example.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<kirby-card [hasPadding]="true" [themeColor]="themeColor">
+  <h1>Above header</h1>
```
Comment (jakobe) [inline src/app/examples/divider-example/divider-example.component.html]: A bit confusing as we also have card headers.... 🤔 
Maybe:
```suggestion
  <h1>Above divider</h1>
```

File: src/app/examples/divider-example/divider-example.component.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+  <kirby-divider></kirby-divider>
+  <p>Below header</p>
```
Comment (jakobe) [inline src/app/examples/divider-example/divider-example.component.html]: A bit confusing as we also have card headers.... 🤔 
Maybe:
```suggestion
  <p>Below divider</p>
```

File: src/app/examples/divider-example/divider-example.component.html (hunk 3 above)
Comment (MortenGregersen) [reply src/app/examples/divider-example/divider-example.component.html]: Typo... :)

File: src/app/examples/divider-example/divider-example.component.html (hunk 4 above)
Comment (MortenGregersen) [reply src/app/examples/divider-example/divider-example.component.html]: Typo... :)

File: src/kirby/components/divider/divider.component.scss (hunk 2 above)
Comment (MortenGregersen) [reply src/kirby/components/divider/divider.component.scss]: I've moved the `:host-context` out to the root level. The `hr` is placed at the root level as it needs the same properties (except from the color) for both contexts.

**Comment by MortenGregersen** [comment]
> LGTM, however you should add the component to the kirby-testing-module so it's being mocked as well :)

Isn't this already done?
https://github.com/kirbydesign/designsystem/pull/732/files#diff-27414d5878458409d9df1ead2db7bbd5R135

**Pull request merged**